src/components/dashboard/InfiniteList.js

```diff
@@ -0,0 +1,29 @@
+import React, {Component, PropTypes} from 'react';
+import InfiniteScroll from 'react-infinite-scroller';
+
+import Loading from './../Loading';
+
+class InfiniteList extends Component {
+    render() {
+        return (
+            <InfiniteScroll
+                pageStart={0}
+                loadMore={this.props.update}
+                hasMore={this.props.items.next}
+                useWindow={false}
+                threshold={-256}
+                loader={<Loading fetching={this.props.items.fetching} length={this.props.items.length}/>}
```

Comment:
Yeah, I wasn't 100% sure which would be best. it's probably worth doing `{..this.props.items}` instead in components that use InfiniteList